packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/internal_utils.ts

```diff
@@ -293,11 +293,5 @@ export function setManaged({
   optionsManaged?: boolean;
   objectManaged?: boolean;
 }): boolean {
-  if (optionsManaged !== undefined) {
-    return optionsManaged;
-  } else if (optionsManaged === undefined && objectManaged !== undefined) {
-    return objectManaged;
-  } else {
-    return false;
-  }
+  return optionsManaged ?? objectManaged ?? false;
```

Comment:
shorthand suggested in https://github.com/elastic/kibana/pull/154515#discussion_r1172309492